### PR TITLE
Complete version of the module 1 Html, Css, Js Capstone Project

### PR DESCRIPTION
**Pull request by oyhpnayiaw**
In this Pull Request, I finished building my capstone project.
- [x] Home Page
- [x] About Page
- [x] Interactive and dynamic contents  
- [x] Desktop Version
- [x] Mobile Version 
- [x] Menu and view more stuff in Mobile Version 
- [x] Finish `README.md` and add acknowledgment of the original author in the file

[Watch Presentation Video](https://www.loom.com/share/0fcb6cf25e814d3c983a2a0f1e26bcd2)

> If you want to run this project in your local environment please follow the *Getting Start* section in the `README.md`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,255 @@
...
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <script src="scripts/menu.js" defer></script>
+    <script src="scripts/voice-actors.js" defer></script>
+  </head>
+  <body class="">
+    <header
```
Comment (codecaiine) [inline index.html:38]: - [ ] Good job so far your navigation is great ... But according to the project requirement, your navbar should be divided into a top dark bar and a bottom white bar. You design the bottom white bar which is impressive :clap: Please make sure that you design the top dark bar in order to make your design complete and great :+1: 

#### Your design
capstone12

#### Expected design
Template12

File: index.html [hunk 2]
```diff
@@ -1,12 +1,255 @@
...
+          </div>
+        </div>
+
+        <button
+          type="button"
+          class="bg-primary py-4 px-6 text-gray-200 font-semibold mt-6 transition md:p-0 hover:bg-opacity-90 md:text-primary md:bg-transparent md:hover:underline"
+        >
+          Join One Piece Fan's Club
+        </button>
+      </section>
```
Comment (codecaiine) [inline index.html:165]: - [ ] Good job so far :clap: But I think there are a few things with your design you need to work on. Please make sure that when the topic card has hovered with the mouse you have a border around it like on the screenshot below and also your button  `Join One Piece Fan's Club` has the same color as on the design. :+1: 

#### Orignal design
Template2

#### Your design
cAPSTONE2

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html:38]: - [ ] @oyhpnayiaw a great job implementing the changes requested however the navbar needs to fill the whole width below its a comparison


Given Design          |  Your design     
:-------------------------:|:-------------------------:
image| image

File: index.html [hunk 3]
```diff
@@ -1,12 +1,264 @@
...
+            </p>
+          </div>
+        </div>
+
+        <button
+          type="button"
+          class="bg-primary py-4 px-6 text-gray-200 font-semibold mt-6 transition md:p-0 hover:bg-opacity-90 md:underline md:text-white md:bg-transparent md:hover:underline"
+        >
+          Join One Piece Fan's Club
+        </button>
```
Comment (danmainah) [inline index.html:173]: - [ ] @oyhpnayiaw great work implementing the changes requested however you forgot to change the join button color to match the design color given.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @oyhpnayiaw , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (oyhpnayiaw) [reply index.html:173]: @danmainah It's already white color as previous review requested.

Comment (oyhpnayiaw) [reply index.html:173]: @danmainah  Can you explain more about this comment?
Screen Shot 2023-01-05 at 10 04 48 PM

File: about.html [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        <div class="text-sm text-white">
+          <p>
+            Copyright (C) 2023 One Piece Project. <br class="md:hidden" />All
+            Rights Reserved.
+          </p>
+          <p class="hidden md:block mt-2 font-light">
+            このサイトは著作権法により保護されており、権利者の許可なく、記事、写真、画像などを無断で使用することはできません。
+          </p>
+        </div>
+      </div>
```
Comment (Qoosim) [inline about.html:189]: - [OPTIONAL] I would recommend you to change your footer background color to white, in order to match what is on the guiding design.
- [OPTIONAL] Also, try to include the partner section that is missing on the about page of your design for mobile version.
-  Please, take a look :point_down: 

### Your Design
your-about-footer

### Guidng Template
temp-about-footer

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @oyhpnayiaw, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

##  Top Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**